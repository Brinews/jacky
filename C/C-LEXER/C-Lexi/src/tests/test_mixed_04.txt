1234567890ABCDEF1234567890ABCDEF x16
1234567890ABCDEF1234567890ABCDEFx16
1234567890ABCDEF1234567890ABCDEFx 16
1234567890ABCDEF1234567890ABCDEFx1 6
1234567890ABCDEF1234567890ABCDEFx1
1234567890ABCDEF1234567890ABCDEFx08
