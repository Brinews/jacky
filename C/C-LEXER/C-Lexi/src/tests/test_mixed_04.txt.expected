{1234567890 , NUM , 1}
{ABCDEF1234567890ABCDEF , ID , 1}
{x16 , ID , 1}
{1234567890ABCDEF1234567890ABCDEFx16 , BASE16NUM , 2}
{1234567890 , NUM , 3}
{ABCDEF1234567890ABCDEFx , ID , 3}
{16 , NUM , 3}
{1234567890 , NUM , 4}
{ABCDEF1234567890ABCDEFx1 , ID , 4}
{6 , NUM , 4}
{1234567890 , NUM , 5}
{ABCDEF1234567890ABCDEFx1 , ID , 5}
{1234567890 , NUM , 6}
{ABCDEF1234567890ABCDEFx08 , ID , 6}
{ , END_OF_FILE , 8}
